### DISCUSSION TITLE
"outdated" French translation

### DISCUSSION THREAD

**Discussion by MarioSwitch** [osmandapp/OsmAnd#21027] [General]
@ldmpub @letypequividelespoubelles 

Ne serait-il pas mieux de remplacer la traduction de "outdated" par "pas à jour" plutôt que "périmé(e)s" ?

J'ai l'impression que ça rend les phrases bizarres ("cartes périmées", comme si c'était à manger 😅) alors que "cartes pas à jour" paraît plus naturel...

(Je vous mentionne parce que vous êtes les 2 autres traducteurs "actifs" de ce que j'ai vu, et j'aimerais avoir votre avis avant de faire la modif')

**Comment by letypequividelespoubelles** [comment]
Ce serait en effet bien mieux 😃

**Reply by ldmpub** [reply]
Ok pour moi. Vous pouvez faire la modif 👌.

**Comment by MarioSwitch** [comment]
@ldmpub @letypequividelespoubelles 

Je viens de modifier les 3 chaînes qui contenaient "périmé", et j'ai ajouté le terme dans le glossaire, histoire qu'on conserve la traduction du terme "outdated" pour de potentielles futures chaînes !
